Marlin/Marlin_main.cpp

```diff
@@ -5688,11 +5688,18 @@ inline void gcode_M77() { print_job_timer.stop(); }
 /**
  * M104: Set hot end temperature
  */
+#if ENABLED(SINGLENOZZLE) || ENABLED(SINGLENOZZLE_SWITCHING_EXTRUDER)
```

Comment:
Indentation of `#if` here should be 2 spaces, with contents indented 2 more spaces.